ngsild-payloads/fluidMonitoring.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+    "id": "urn:ngsi-ld:FluidMonitoring:AAA-111-CCC",
+    "type": "FluidMonitoring",
+    "refDevice": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:Device:AAA-111-CCC"
+    },
+    "fluidType": {
+        "type": "Property",
+        "value": "Electricity"
+    },
+    "consumption": {
+        "type": "Property",
+        "value": 123,
+        "unitCode": "KWH",
+        "observedAt": "2023-12-14T15:00:00Z"
+    }
```

Comment:
I think we need to be a bit more specific here since the entity is generic. 
So we could have properties:
- waterConsumption
- electricityConsumption
- gasConsumption

Reply:
I though the `fluidTypes` property was enough but why not :)

Reply:
Ah ok so it means there are as many FluidMonitoring entities as there are fluids ? Ok I did not understand at first sight. It works well as well actually

Reply:
Yes, as many entities as there are fluids. This also allows them to evolve independently (and with fluid specific properties).